packages/api-clients/open-api/m2mGatewayApi.yml

```diff
@@ -2292,6 +2292,19 @@ paths:
               $ref: "#/components/headers/RateLimitRemainingHeader"
             X-Rate-Limit-Interval:
               $ref: "#/components/headers/RateLimitIntervalHeader"
+        "409":
```

Comment:
Missing also the 403 Forbidden that can be returned by the process or m2m-gw in case of non-admin